src/pages/Account/Tabs/ModulesTab.tsx

```diff
@@ -1,5 +1,13 @@
 import {Types} from "aptos";
-import {Box, Grid, Stack, Typography, useTheme} from "@mui/material";
```

Comment:
https://user-images.githubusercontent.com/121921928/224185381-a33ce61f-b805-4861-80ae-a9066c9b7a49.mov

LGTM! great manipulation on modules and functions. 

one general nit is that when i refresh the page, it will redirect me back to the initial stage where "view code" selected. would love to keep stay on the same tab and same function's selected as well.

Reply:
that's where URL helps.

For example:

```
/account/[account]/modules/code/?module=[module]
```

and

```
/account/[account]/modules/write/?module=[module]&fn=[fn]
```

otherwise need to store info in local storage, which I think less optimal.

however, this routing change would be a larger scope change...

thoughts? @jillxuu @0xZihan

Reply:
URL makes sense to me.

Reply:
believe this kind of routing change is out of scope - can we worry about it later?

Reply:
add url param strings is out of scope, but adding tabs in the param should be in scope so that when tab switching, url will change. 

https://user-images.githubusercontent.com/121921928/224802942-da90ac29-66a5-49a8-95fd-6568f872eb54.mov

Reply:
you could add something like `<Route path=":address/:tab/:action" element={<AccountPage />} />`  to the `/account` routing, and read the action from the param like `const {action} = useParams();`

Reply:
@jillxuu this maybe find for now, but `:address/:tab/:action` seems too general 🤔 it seems more appropriate to add `:/address/modules/:tab` routes? wdyt

Reply:
yeah this makes sense cuz it's stricter routing wise. feel free to go with this approach